### DISCUSSION TITLE
Disabled button disappear when hover

### DISCUSSION THREAD

**Discussion by onlyhasbi** [chakra-ui/chakra-ui#7910] [Q&A]
how to avoid the disabled button not disappearing when hovering ?

note : i have a custom button
`components: {
    Button: {
      variants: {
        primary: {
          bg: 'brand.100',
          color: 'white',
          _hover: {
            bg: 'brand.100',
            opacity: 0.9,
          },
        },
      },
    },
  },`

**Comment by jansengunderson** [comment]
It looks like the default disabled hover state is to set the button's background color to white. I was able to fix it like this:
```
   Button: {
      variants: {
        "primary": {
          backgroundColor: "brand.500",
          color: "white",
          borderRadius: "sm",
          _hover: {
            backgroundColor: "brand.200",
            color: "white",
          },
          _disabled: {
            _hover: {
              backgroundColor: "brand.200 !important",
              color: "white !important",
            },
          },
        },
      },
    }
```
